layouts/default.vue

```diff
@@ -111,6 +111,12 @@ export default Vue.extend({
           href: 'https://api.mapbox.com/mapbox-gl-js/v1.8.1/mapbox-gl.css'
         }
       ],
+      titleTemplate:
+        '%s | ' +
+        this.$t('東京都') +
+        ' ' +
+        this.$t('新型コロナウイルス感染症') +
+        this.$t('対策サイト'),
```

Comment:
[テンプレートリテラル](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/template_strings)を使うと、ソースコードから出力内容が分かりやすくなります。

```suggestion
      titleTemplate: `%s | ${this.$t('東京都')} ${this.$t('新型コロナウイルス感染症'} ${this.$t('対策サイト')}`
```